continuousflex/protocols/utilities/deep_hemnma.py

```diff
@@ -60,8 +61,21 @@ def train(imgs_path, output_path, epochs=400, batch_size=2, lr=1e-4, flag=0, dev
         DEVICE = 'cpu'
     mean, std = norm(imgs_path, output_path, FLAG, mode, batch_size)
     transform = transforms.Compose([transforms.ToTensor(), transforms.Normalize((mean), (std))])
-    dataset = cryodata(imgs_path, output_path, flag=FLAG, mode = mode, transform=transform)
-    dataset_size = len(dataset)
+    transform1 = transforms.Compose([transforms.ToTensor(),
+                                    transforms.RandomRotation((-45, 45)),
```

Comment:
You are augmenting the dataset for training, I am right? If so, 45 degrees for single particle images might cause severe interpolation errors since the images are too small compared to natural images. I would suggest keeping with 90 180 270 only.
Keep in mind for later if this method is to be extended to tomography that no rotation is allowed since it causes rotation of the missing wedge region.